src/Compilers/CSharp/Portable/Symbols/AssemblySymbol.cs

```diff
@@ -494,6 +496,21 @@ internal bool RuntimeSupportsByRefLikeGenerics
             }
         }
 
+#nullable enable
+        private bool _overrideRuntimeSupportsAsync;
+#pragma warning disable VSTHRD200 // Use "Async" suffix for async methods
```

Comment:
nit: instead of suppressing warning, consider renaming to "...SupportsAsyncMethods" (which matches the `RuntimeCapability` name)